tasks/stats/processing.go

```diff
@@ -126,7 +128,7 @@ func (r *ProcessingStatsRefresher) Run(ctx context.Context) error {
 }
 
 func (r *ProcessingStatsRefresher) collectStats(ctx context.Context) (bool, error) {
-	subQueries := []string{statsActors}
+	subQueries := []string{fmt.Sprintf(statsActors, actorCodeCase)}
```

Comment:
does this mean we're templating the case statement into each of the sub-select's in the union?
is there a way to select that into a place that can then be directly used in each of the union'd selects directly?

Reply:
I could try something clever like inserting into a temporary table and joining against it, but I'm not sure it's worth it. The case is evaluated once per actor code per subquery (since we have already grouped by code). Perhaps about 120 times which I think is quite low compared to the cost of grouping over 30 million rows.